spec/base_spec_helper.rb

```diff
@@ -102,8 +102,12 @@
   # Take example responses from Rswag specs for API documentation.
   # https://github.com/rswag/rswag#enable-auto-generation-examples-from-responses
   config.after(:each, :rswag_autodoc) do |example|
+    # Categories and group operations of the same API endpoint.
+    example.metadata[:operation][:tags] ||= [self.class.top_level_description]
+
```

Comment:
These first few commits are in a previous PR:

* #11494

You can ignore them and start at:

* [Add DFC API EnterpriseGroups#show](https://github.com/openfoodfoundation/openfoodnetwork/pull/11508/commits/99234b957b14d0930bcbf05f9de148379b67b4fe)